lib/components/Button/Button.docs.tsx

```diff
@@ -81,9 +175,10 @@ const docs: ComponentDocs = {
           </Text>
         </>
       ),
+      background: 'brand',
       Example: () =>
         source(
-          <Box background="brand" padding="large" borderRadius="standard">
+          <Box background="brand">
```

Comment:
This `Box` can be removed now, think that was the plan behind the change anyway.

Reply:
I left this in for Playroom, but I used the `background` property on the example object so that the gutter matches the other examples. Might be a good idea to automatically add this Box to Playroom links when the example has a background?

Reply:
Ah gotcha. Yeah that may be a nice update. But dont stress about coupling the two